[2022-07-28 09:51:19,510] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v4.upload_to_s3_2020-05 manual__2022-07-28T09:51:01.397439+00:00 [queued]>
[2022-07-28 09:51:19,561] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v4.upload_to_s3_2020-05 manual__2022-07-28T09:51:01.397439+00:00 [queued]>
[2022-07-28 09:51:19,562] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 09:51:19,562] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-28 09:51:19,562] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 09:51:19,624] {taskinstance.py:1397} INFO - Executing <Task(LocalFilesystemToS3Operator): upload_to_s3_2020-05> on 2022-07-28 09:51:01.397439+00:00
[2022-07-28 09:51:19,638] {standard_task_runner.py:52} INFO - Started process 1462 to run task
[2022-07-28 09:51:19,671] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataquality_pipeline_v4', 'upload_to_s3_2020-05', 'manual__2022-07-28T09:51:01.397439+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/dataquality_pipeline_v4.py', '--cfg-path', '/tmp/tmp6_6svgxo', '--error-file', '/tmp/tmp_7k8jprz']
[2022-07-28 09:51:19,675] {standard_task_runner.py:80} INFO - Job 202: Subtask upload_to_s3_2020-05
[2022-07-28 09:51:19,921] {task_command.py:371} INFO - Running <TaskInstance: dataquality_pipeline_v4.upload_to_s3_2020-05 manual__2022-07-28T09:51:01.397439+00:00 [running]> on host 2c2de32fb47f
[2022-07-28 09:51:20,273] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=abin404@yopmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataquality_pipeline_v4
AIRFLOW_CTX_TASK_ID=upload_to_s3_2020-05
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T09:51:01.397439+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-28T09:51:01.397439+00:00
[2022-07-28 09:51:20,314] {base.py:68} INFO - Using connection ID 'aws-conn' for task execution.
[2022-07-28 09:51:20,316] {base_aws.py:206} INFO - Credentials retrieved from login
[2022-07-28 09:51:21,663] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataquality_pipeline_v4, task_id=upload_to_s3_2020-05, execution_date=20220728T095101, start_date=20220728T095119, end_date=20220728T095121
[2022-07-28 09:51:21,745] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 09:51:22,058] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
